lib/spdy-transport/protocol/spdy/parser.js

```diff
@@ -172,7 +173,7 @@ Parser.prototype.onSynHeadFrame = function onSynHeadFrame(type,
 
     if (!isPush) {
       callback(null, {
-        type: 'HEADERS',
+        type: 'SYN_STREAM',
```

Comment:
The framer has them apart: `requestFrame`, `responseFrame`, `headersFrame`. The particular implementation chooses the right frame type.